ts/editor/NoteEditor.svelte

```diff
@@ -572,11 +538,14 @@ the AddCards dialog) should be implemented in the user of this component.
         </Absolute>
     {/if}
 
-    <!-- for init maskeditor -->
-    <div id="io-mask-editor" style="display: none;" />
+    {#if imageOcclusionMode && isImageOcclusion}
+        <div style="display: {$showMaskEditor ? 'block' : 'none'}">
+            <ImageOcclusionPage mode={imageOcclusionMode} />
+        </div>
+    {/if}
 
-    {#if isShowMaskEditor && isImageOcclusion && !isIOImageLoaded}
-        <div id="io-select-image-div" style="padding-top: 60px; text-align: center">
+    {#if $showMaskEditor && isImageOcclusion && !isIOImageLoaded}
```

Comment:
Minor: $ioMaskEditorVisible would be slightly more clear, as 'show' is a verb and sounds like an action.